django-stubs/db/models/base.pyi

```diff
@@ -22,7 +22,7 @@ class Model(metaclass=ModelBase):
     class Meta: ...
     _meta: Options[Any]
     _default_manager: BaseManager[Model]
-    objects: BaseManager[Any]
+    objects: ClassVar[BaseManager[Any]]
```

Comment:
This is a great addition! But, this does raise an important topic to discuss.

Currently, we do not treat properties as `ClassVar`s. I guess that we need to fix that. And mark ones that apply as `ClassVar`s.

@danifus can you please open a new issue?

Reply:
Hi, was this comment intended for me or @davidhalter?

Reply:
Oups! 😆 My bad!

But, feel free to participate!

Reply:
@sobolevn I don't really understand what needs to be fixed. I see that the CI is still failing even after I force pushed an amended commit. I'm guessing that has something to do with the Mypy plugin. But since I'm not using Mypy, I'm only interested in correct type annotations.

I obviously understand that this is more work, but it's not my forte. So I would appreciate help to get this merged. As a temporary measure, I might just ship a patched version in Jedi until this gets fixed.

Still thanks for looking at it @sobolevn!

Reply:
@sobolevn Failing tests are not a matter of the bug I've fixed, I've just checked that. Introducing ClassVars to our stubs it quite a big change. It'd be awesome @davidhalter if you could create an issue on this one. As @sobolevn wrote, marking properties that apply as `ClassVar` would be a lot of help too. Thank you for your work, I can take it from here if you want me to 😄

Reply:
@kszmigiel Yeah feel free to take it from here, I really have no idea :)

I can of course create an issue, but I'd rather not, since I really don't understand what the issue is. The issue would probably have the content "ClassVar for `Model.objects` doesn't work". If that has value for you, I'm fine doing that :).

Reply:
@davidhalter Sure thing, I'll create an issue for that later, thanks for pointing this out 😄